src/index.js

```diff
@@ -0,0 +1,159 @@
+"use strict";
+
+document.addEventListener("DOMContentLoaded", function() {
+
+let currentTemp = 72;
+let sky = "";
+// let defaultCityName = "What city are you searching for?";
+let defaultCityName = "";
+
+
+function getTemp() {
+    const temperature = document.getElementById("temp");
+    temperature.textContent = currentTemp;
+    if (currentTemp >= 80) {
+        temperature.className = "hot";
+    } else if (currentTemp >= 70) {
+        temperature.className = "warm";
+    } else if (currentTemp >= 60) {
+        temperature.className = "medium";
+    } else if (currentTemp >= 50) {
+        temperature.className = "cool";
+    } else {
+        temperature.className = "cold";
+    }
+}
+
+const button1 = document.getElementById("increase_temperature");
+button1.addEventListener("click", () => {
+    currentTemp = currentTemp + 1;
+    getTemp();
+    getLandscape();
+});
+
+const button2 = document.getElementById("decrease_temperature");
+button2.addEventListener("click", () => {
+    currentTemp = currentTemp - 1;
+    getTemp();
+    getLandscape();
+});
+
+const getLandscape = () => {
+    const landscape = document.getElementById("landscape");
+    if (currentTemp >= 80) {
+        landscape.textContent = "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂"
+    } else if (currentTemp >= 70) {
+        landscape.textContent = "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷"
+    } else if (currentTemp >= 60) {
+        landscape.textContent = "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃"
+    } else if (currentTemp >= 50) {
+        landscape.textContent = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲"
+    } else {
+        landscape.textContent = "⛄️⛄️🎁🎿🎿⛄️🦌🎿🎁⛄️⛄️🦌🦌⛄️🎁"
+    }
+}
+
+const getSky = () => {
+    const skyElement = document.getElementById('sky');
+    const skyChoice = document.getElementById('sky-select').value
```

Comment:
This variable is not being used and can be removed since you're storing the sky choice in the `sky` global variable.